rotkehlchen/globaldb/handler.py

```diff
@@ -135,6 +136,22 @@ def __new__(
         GlobalDBHandler.__instance.conn = _initialize_global_db_directory(data_dir, sql_vm_instructions_cb)  # noqa: E501
         return GlobalDBHandler.__instance
 
+    @staticmethod
+    def packaged_db_conn() -> DBConnection:
+        """Return a DBConnection instance for the packaged global db."""
+        if GlobalDBHandler() is not None:
+            if GlobalDBHandler()._packaged_db_conn is not None:
+                # mypy does not recognize the initialization as that of a singleton
+                return GlobalDBHandler()._packaged_db_conn  # type: ignore
+
+        packaged_db_path = Path(__file__).resolve().parent.parent / 'data' / 'global.db'
+        GlobalDBHandler()._packaged_db_conn = initialize_globaldb(
+            dbpath=packaged_db_path,
+            sql_vm_instructions_cb=DEFAULT_SQL_VM_INSTRUCTIONS_CB,
+        )
+        # mypy does not recognize the initialization as that of a singleton
+        return GlobalDBHandler()._packaged_db_conn  # type: ignore
```

Comment:
maybe just to make it clean and avoid the type ignore

```suggestion
        packaged_db_conn = initialize_globaldb(
            dbpath=packaged_db_path,
            sql_vm_instructions_cb=DEFAULT_SQL_VM_INSTRUCTIONS_CB,
        )
        GlobaldbHandler()._packaged_db_conn = packaged_db_conn
        return packaged_db_conn
```